app/src/main/kotlin/com/mapzen/erasermap/model/LostClientManager.kt

```diff
@@ -44,7 +44,7 @@ class LostClientManager(val application: Application, locationFactory: LocationF
   }
 
   override fun connect() {
-    if (state == State.CONNECTING) {
+    if (state == State.CONNECTING || state == State.CONNECTED) {
```

Comment:
This is going to conflict with https://github.com/mapzen/eraser-map/pull/750 where I removed tracking the connection state locally in favor of calling `lostClient.isConnected()`. Would be be able check the state of the client here instead?